system/languages/en.php

```diff
@@ -248,14 +248,15 @@
 		"ml-mail-has-invalid-status" => "this email has an invalid status",
 		"ml-invalid-command" => "invalid command",
 		"ml-inconsistent-parameters" => "the given parameters are inconsistent",
-		"ml-successfully-subscribed" => "successfully subscribed",
-		"ml-successfully-unsubscribed" => "successfully unsubscribed",
+		"ml-successfully-subscribed" => "you have successfully subscribed",
+		"ml-successfully-unsubscribed" => "you have successfully unsubscribed",
 		"ml-successfully-created" => "successfully created",
 		"ml-successfully-updated" => "successfully updated",
-		"ml-successfully-sent" => "successfully sent",
+		"ml-successfully-sent" => "You will soon receive an email with a link. Please confirm your subscription. If you do not get the email, it may have landed in your spam folder.",
```

Comment:
This changes the meaning of this string significantly. I would suggest we also rename it, to something like `ml-please-confirm-by-email` or something?

Reply:
renamed